src/components/views/rooms/wysiwyg_composer/WysiwygComposer.tsx

```diff
@@ -53,18 +57,8 @@ export function WysiwygComposer(
 
     return (
         <div className="mx_WysiwygComposer">
-            <div className="mx_WysiwygComposer_container">
-                <div className="mx_WysiwygComposer_content"
-                    ref={ref}
-                    contentEditable={!disabled && isWysiwygReady}
-                    role="textbox"
-                    aria-multiline="true"
-                    aria-autocomplete="list"
-                    aria-haspopup="listbox"
-                    dir="auto"
-                    aria-disabled={disabled || !isWysiwygReady}
-                />
-            </div>
+            <FormattingButtons wysiwyg={wysiwyg} formattingStates={formattingStates} />
```

Comment:
Probably a minor nitpick, and feel free to punt that comment to a later stage, but rather than naming props `wysiwyg` I think we should call them `composer`.

It's only a temporary thing that we have two different composer in the codebase, the ultimate goal would be to have this one taking over, and when that happens we will end up having a `wysiwyg` prop that's hard to parse.

What do you think?

Reply:
I agree with your idea, I'll rename it!